e2e/testsuite/grpc_query.go

```diff
@@ -73,6 +73,7 @@ func (s *E2ETestSuite) InitGRPCClients(chain *cosmos.CosmosChain) {
 
 	s.grpcClients[chain.Config().ChainID] = GRPCClients{
 		ClientQueryClient:      clienttypes.NewQueryClient(grpcConn),
+		ConnectionQueryClient:  connectiontypes.NewQueryClient(grpcConn),
```

Comment:
the wonders of allowing default initialization.

Reply:
I just stumbled upon this error when running the v7.1 upgrade test. :)

Reply:
Multiple victims of this unforgiving language design